src/main/java/com/jvm_bloggers/frontend/public_area/blogs/BlogPostsPage.java

```diff
@@ -0,0 +1,81 @@
+package com.jvm_bloggers.frontend.public_area.blogs;
+
+import com.jvm_bloggers.entities.blog.BlogRepository;
+import com.jvm_bloggers.entities.blog_post.BlogPost;
+import com.jvm_bloggers.entities.blog_post.BlogPostRepository;
+import com.jvm_bloggers.frontend.admin_area.PaginationConfiguration;
+import com.jvm_bloggers.frontend.common_components.infinite_scroll.InfinitePaginationPanel;
+import com.jvm_bloggers.frontend.common_components.request_handlers.BlogPostsPageRequestHandler;
+import com.jvm_bloggers.frontend.public_area.AbstractFrontendPage;
+
+import org.apache.wicket.markup.html.WebMarkupContainer;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.apache.wicket.markup.repeater.Item;
+import org.apache.wicket.markup.repeater.data.DataView;
+import org.apache.wicket.request.mapper.parameter.PageParameters;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wicketstuff.annotation.mount.MountPath;
+
+import static com.jvm_bloggers.frontend.public_area.blogs.BlogPostsPage.BLOG_ID_PARAM;
+import static com.jvm_bloggers.utils.DateTimeUtilities.DATE_FORMATTER;
+
+@MountPath("blogs/${" + BLOG_ID_PARAM + "}/posts")
```

Comment:
After looking on `blog` table I did not come up with any reasonable pattern to create a unique identifier for blog base on name or author. Here are some examples that make this hard (it is not always first and second name):
```
Anna Pietras, Jakub Derda                | http://kobietydokodu.pl
JavaCzyHerbata.pl                        | http://javaczyherbata.pl
Tomek (TJ) Janczewski                    | http://3pio.pl/
Konrad 'ktoso' Malawski                  | http://malaw.ski/
```

after all I will use `jsonId` and find that other task to make it obsolete.